bridge-sdk/connectors/omni-connector/src/omni_connector.rs

```diff
@@ -262,7 +262,9 @@ impl OmniConnector {
             token: message_payload.token.to_string(),
             amount: message_payload.amount.into(),
             recipient: match message_payload.recipient {
-                OmniAddress::Eth(addr) => H160(addr.0),
+                OmniAddress::Eth(addr) | OmniAddress::Base(addr) | OmniAddress::Arb(addr) => {
```

Comment:
Should we add a method `is_evm_address` to OmniAddress?

Reply:
How it will help? We need something that would convert evm to h160, but this is a refactor task and it can be done in the next PR in my opinion

Reply:
Okay, but still this is not related to this PR. We would need to introduce `is_evm...` and `get_evm_address` as a method for `OmniAddress` struct